### ISSUE TITLE
iOS build with Xcode 8 throws "Command xcodebuild failed with exit code 65"

### ISSUE THREAD

**Issue by NickIliev** [bradmartin/nativescript-cardview#29]
Using the plugin to build a project with Xcode 8 is throwing due to:

```
=== BUILD TARGET MaterialCard OF PROJECT Pods WITH CONFIGURATION Debug === 

“Use Legacy Swift Language Version” (SWIFT_VERSION) is required 
to be configured correctly for targets which use Swift. 
Use the [Edit > Convert > To Current Swift Syntax…] menu to choose a Swift
version or use the Build Settings editor to configure the build setting directly. 
```

And the project build via Xcode returns error:

```
ld: framework not found MaterialCard
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

**Comment by NickIliev** [comment]
The [pod file MaterialCard](https://github.com/NathanWalker/MaterialCard) should be converted to SWIFT 3.0 and updated on the CocoaPods site in order for this plugin to work with XCode 8.

**Comment by bradmartin** [comment]
Thanks Nick. I have no clue about this iOS jazz :) I'll see if Walker can
fix that

**Comment by NickIliev** [comment]
I was able to locally convert ti to Swift 3.0 and apply it to my test project the following way.
- Open the Pod in XCode 8 ( _Pods.xcodeproj file )
- Convert to Swift 3.0 (will prompt right away or Edit > Convert >> To Currrent Swift Syntax)

Now to apply the converted pod I had to add this in the **plugin** Podfile

```
pod 'MaterialCard', :local => "/Users/niliev/Desktop/test/MaterialCard" // I've changed this only to use my local coverted pod 

post_install do |installer|
  installer.pods_project.targets.each do |target|
    target.build_configurations.each do |config|
      config.build_settings['SWIFT_VERSION'] = '3.0'
    end
  end
end
```

Still I am not pretty sure if this way will have backwards compatibility or maybe there is a most human way to force convertion

**Comment by bradmartin** [comment]
@NathanWalker  ^^^

**Comment by hypery2k** [comment]
as a workaround you can add this to `app/AppResources/iOS/build.xconfig`:

```
...
SWIFT_VERSION = 2.3
```

**Comment by NathanWalker** [comment]
closes via https://github.com/bradmartin/nativescript-cardview/pull/31

Published Swift 3.0 compatibility with `1.2.0` 👍
